KubeArmor/monitor/logUpdate.go

```diff
@@ -214,6 +214,41 @@ func (mon *SystemMonitor) UpdateLogs() {
 				log.Resource = "domain=" + sockDomain + " type=" + sockType + " protocol=" + getProtocol(sockProtocol)
 				log.Data = "syscall=" + getSyscallName(int32(msg.ContextSys.EventID))
 
+			case TCPConnect, TCPConnectv6:
+				if len(msg.ContextArgs) != 2 {
+					continue
+				}
+				var sockAddr map[string]string
+				var protocol string
+				if val, ok := msg.ContextArgs[0].(string); ok {
+					protocol = val
+				}
+
+				if val, ok := msg.ContextArgs[1].(map[string]string); ok {
+					sockAddr = val
+				}
+
+				log.Operation = "Network"
+				log.Resource = "remoteip=" + sockAddr["sin_addr"] + " port=" + sockAddr["sin_port"] + " protocol=" + protocol
+				log.Data = "kprobe=tcp_connect domain=" + sockAddr["sa_family"]
+
+			case TCPAccept, TCPAcceptv6:
```

Comment:
This seems almost duplicate of TCPConnect/6 above. Can we refactor the code?

Reply:
Done